data/human/boarding missions.txt

```diff
@@ -532,7 +532,7 @@ mission "Slave Crew Rescue"
 				launch
 
 			label "has brig"
-			`        You search Tarquin and find a concealed handgun and several knives, the credit chip he was going to pay you, and a tiny unlocking device. Then you instruct one of the members of your boarding party to lock him in your ship's brig. Tarquin curses you foully as he is marched to captivity at gunpoint. You resolve to figure out what to do with him later, and go to rescue the ship's crew.`
+			`		You search Tarquin and find a concealed handgun and several knives, the credit chip he was going to pay you, and a tiny unlocking device. Then you instruct one of the members of your boarding party to lock him in your ship's brig. Tarquin curses you foully as he is marched to captivity at gunpoint. You resolve to figure out what to do with him later, and go to rescue the ship's crew.`
```

Comment:
Shouldn't this just be one tab?

Reply:
Not sure; my editor made it look like it was two tabs wide, but it looks like a single tab on github. I think there are 8 spaces, so that should be two tabs in most cases.

Reply:
I don't think it matters if it had 8 spaces or not, does it? There should always just be a single tab to start off a paragraph, so far as I know.

Reply:
The first line of the conversation should have no indent. Everything else should be 1 tab.
Looking at the conversation this is in, it isn't the first line so it should be indented, but I see no reason ffor it to be indented two tabs.
```suggestion
			`	You search Tarquin and find a concealed handgun and several knives, the credit chip he was going to pay you, and a tiny unlocking device. Then you instruct one of the members of your boarding party to lock him in your ship's brig. Tarquin curses you foully as he is marched to captivity at gunpoint. You resolve to figure out what to do with him later, and go to rescue the ship's crew.`
```